api/repositories/service_offering_repository_test.go

```diff
@@ -28,6 +28,110 @@ var _ = Describe("ServiceOfferingRepo", func() {
 		)
 	})
 
+	Describe("Get", func() {
```

Comment:
Could you add a test for the cases when getting the offering fails? For example, test that getting an offering with a dummy ID results into an error

Reply:
Done